.github/workflows/tests.yml

```
@@ -38,4 +37,25 @@ jobs:
         run: |
             python -m pip install --verbose -e .
       - run: |
-            make test BTEST=`which btest`
+            make test BTEST=$(which btest)
+
+  upload:
+    runs-on: ubuntu-latest
+    needs: [Run-BTest]
+    if: github.repository == 'zeek/pysubnettree' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check release version
+        # This fails e.g. if VERSION contains a dev commits suffix,
+        # since we don't want to push these to PyPI. Accepts two-
+        # and three-component version numbers (e.g. 1.0 and 1.0.1).
+        run: |
+          grep -E -x '[0-9]+\.[0-9]+(\.[0-9]+)?' VERSION
+      - name: Build wheel
```

```suggestion
      - name: Build sdist
```
since `make dist` actually does a `python3 setup.py sdist`. This is different from zkg, where we use `bdist_wheel`.